### PR TITLE
Fixed #26039 -- Unwrapped nested partials in URL reversal.

### PR DESCRIPTION
**Pull request by gst**

### CODE REVIEW

**Comment by gst** [comment]
fixing isort problems..

File: tests/urlpatterns_reverse/urls.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from tests.urlpatterns_reverse.views import empty_view_nested_partial
```
Comment (timgraham) [inline tests/urlpatterns_reverse/urls.py]: This should be in the existing `from .views` block.

**Comment by timgraham** [comment]
You may squash commits.

File: tests/urlpatterns_reverse/urls.py (hunk 1 above)
Comment (gst) [inline tests/urlpatterns_reverse/urls.py]: damn, effectively, doing that

**Comment by gst** [comment]
> You may squash commits.

I've done that so that one can checkout at the test commit, run the test and see the failure/bug, then checkout the fix commit and see that it effectively fixes the bug..

you think it's better to have the test as well as the fix in the same commit ?  I don't think so but if you prefer that then I'll squash them..

**Comment by timgraham** [comment]
Yes, it's fine to squash. I can check for a proper regression test using `git checkout HEAD~1 django/urls/resolvers.py`.

**Comment by gst** [comment]
here it is :)

**Comment by timgraham** [comment]
buildbot, add to whitelist.

**Comment by timgraham** [comment]
merged in 9f9921e89c5c559ff4324f8cf968007cefbf6442, thanks!

**Comment by gst** [comment]
Django is welcome ;)

**Pull request closed without merging**